### PR TITLE
feat(manifests,rootfs): add nsqlookupd to the mix

### LINKED ISSUE #3: Add a chart for nsqlookupd
`nsqlookupd` will be important for us if we want to add more `nsqd` instances - otherwise known as horizontally scaling.

See http://nsq.io/components/nsqlookupd.html for documentation on `nsqlookupd`
# Notes on horizontally scaling our queueing subsystem

Note that, as of this writing, I have been able to run 4000 concurrent instances of this command:

``` console
boom -n 1000 -c 500 ...
```

In english that test produced 4,000,000 total log messages with up to 2,000,000 of them concurrently entering the queue. A single `nsqd` pod was able to handle that specific workload.

This test was run on a Google Container Engine cluster of 10 `n1-standard-2 (2 vCPUs, 7.5 GB memory)` nodes. 

When I ran 5000 concurrent instances of the above `boom` command, the nsq crashed.

### PR DESCRIPTION
**Pull request by arschles**
Fixes #3 

This PR is intended to do the following:
- Add [nsqlookupd](http://nsq.io/components/nsqlookupd.html) to the Docker image
- Add nsqlookupd manifests
- Add start script for nsqlookupd
- Connect nsqd instances to the lookupd in the `start-nsqd` script
- Only run a single `nsqlookupd` instance, not multiple

### CODE REVIEW

File: rootfs/opt/nsqlookupd/bin/start-nsqlookupd [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+export MAX_READY_COUNT=${MAX_READY_COUNT:-10000}
+export BEARER_TOKEN=/var/run/secrets/kubernetes.io/serviceaccount/token
+export DATA_PATH=${DATA_PATH:-/opt/nsqlookupd/data}
+if [ -f $BEARER_TOKEN ]; then
+    export TOKEN=$(cat $BEARER_TOKEN)
+    export POD_API_URL=https://$KUBERNETES_SERVICE_HOST:$KUBERNETES_SERVICE_PORT/api/v1/namespaces/$POD_NAMESPACE/pods/$HOSTNAME
+    export NODE_NAME=$(curl -s $POD_API_URL --header "Authorization: Bearer $TOKEN" --insecure | grep nodeName | cut -c 18- | tr -d '",')
+    echo "Set broadcast-address to ${NODE_NAME} and MAX_READY_COUNT to ${MAX_READY_COUNT}"
+    exec nsqlookupd -http-address="0.0.0.0:$NSQLOOKUPD_HTTP_PORT" -tcp-address="0.0.0.0:$NSQLOOKUPD_TCP_PORT"
```
Comment (jchauncey) [inline rootfs/opt/nsqlookupd/bin/start-nsqlookupd:11]: dont http-address and tcp-address default to 0.0.0.0?

Comment (arschles) [inline rootfs/opt/nsqlookupd/bin/start-nsqlookupd:11]: yea, but I wanted to explicitly set the port

**Comment by jchauncey** [comment]
@arschles do we just wnat to close this for now?

**Comment by aegixx** [comment]
What's the status of this?  I'm open to assisting if someone can identify where the issues are.

**Comment by jchauncey** [comment]
There isnt a real need for multiple nsqs right now so having an nsqlookupd process just doesnt make sense. I'm going to close this for now.

**Pull request closed without merging**